hummingbot/connector/exchange/kraken/kraken_exchange.pyx

```diff
@@ -107,11 +110,13 @@ cdef class KrakenExchange(ExchangeBase):
                  kraken_secret_key: str,
                  poll_interval: float = 30.0,
                  trading_pairs: Optional[List[str]] = None,
-                 trading_required: bool = True):
+                 trading_required: bool = True,
+                 kraken_api_tier: str = "starter"):
```

Comment:
It might be better to use the Enum values here (transform the info to an Enum component right after loading the configuration)